src/utils/helpers.ts

```diff
@@ -1,4 +1,12 @@
+import { InstitutionName } from './../types/institution.types';
+import i18n from '../translations/i18n';
+
 // Method that retrieves the value of a (potentially) dotted keyString from a object
 export const getObjectValueByFieldName = (object: any, keyString: string) => {
   return keyString.split('.').reduce((obj, key): any => obj && obj[key], object);
 };
+
+// Method that retrieves the name of an institution/organization based on selected language
```

Comment:
synes vi burde hatt egne helper-filer for ting som ikke er fullstendig generiske. en institution-helper.fex ? Kan fort bli mye rart i en helper-fil. Funksjonsnavnet burde i det minste spesifisere at det er institusjoner det dreier seg om.

Reply:
Endrer navnet. Foreløpig er det 2 metoder i denne fila så syns det er unødvendig å ta ting ut før det blir et problem med mange metoder.